app/src/cambodia/java/org/eyeseetea/malariacare/domain/usecase/strategies/GetReviewValuesBySurveyIdUseCaseStrategy.java

```diff
@@ -0,0 +1,46 @@
+package org.eyeseetea.malariacare.domain.usecase.strategies;
+
+import org.eyeseetea.malariacare.data.database.model.Option;
+import org.eyeseetea.malariacare.data.database.utils.Session;
+import org.eyeseetea.malariacare.domain.boundary.executors.IAsyncExecutor;
+import org.eyeseetea.malariacare.domain.boundary.executors.IMainExecutor;
+import org.eyeseetea.malariacare.domain.boundary.repositories.IValueRepository;
+import org.eyeseetea.malariacare.domain.entity.Value;
+import org.eyeseetea.malariacare.domain.usecase.AGetReviewValuesBySurveyIdUseCaseStrategy;
+import org.eyeseetea.malariacare.layout.adapters.survey.navigation.NavigationController;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class GetReviewValuesBySurveyIdUseCaseStrategy extends
```

Comment:
I think that we have a mix of nomenclature about when we assign a name as strategy and when not. I think that root element should be named without strategy in name as LoginActivity, LoginUseCase and then if this root element has little different behaviors between variants then create strategies as ALoginActivityStrategy, IPullDhisSDKDataSourceStrategy that it's invoked from root element but if the behavior difference is big then create a base abstract class as aLoginUseCase in main if there are common code and derivate class in every variant as LoginUseCase but root element should have not strategy in name. Can you refactor this use case according to this? thanks